wayland-scanner/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Update `quick-xml` to 0.37
```

Comment:
This seems strange - was updating this file overlooked when cutting the last release? :thinking:

Reply:
Yeah. I guess I missed updating that file in https://github.com/Smithay/wayland-rs/pull/829.

(Other aspects of the release process has automation and checks, but not the changelog files.)